src/RBACHandler.ts

```diff
@@ -45,6 +46,20 @@ export class RBACHandler implements Authorization {
         return authZResponses.every(Boolean);
     }
 
+    getAllowedResourceTypesForOperation(request: getAllowedResourceTypesForOperationRequest): string[] {

```

Comment:
I wonder if it would be better to return a set instead of a list? quicker look up on the other side, thoughts?

Reply:
I think an array is better for the interface. 

Sets were introduced in ES6 so they don't have widespread use (I think). I don't recall any npm package w functions that use sets as arguments/return types.

The other thing is that sets do not have a JSON representation. This makes them cumbersome to work with for things like http calls since you need to convert them back and forth to arrays anyways.

Sets do have fast lookups though, but If your code is doing lots of lookups you can always first build a Set from the array (our search implementation for revinclude does that)